### PR TITLE
Added DeleteIndexableRelationsInterface to be able to delete related entities in solr

### PR DESCRIPTION
**Pull request by cynthiameiring**
In case of cascade delete, related entities need to be deleted in Solr.
(Created together with @7ochem )

Before merging, the CHANGELOG will be updated

### CODE REVIEW

**Review by wtfzdotnet — DISMISSED** [review]
Interface komt niet overeen met implementatie, last minute refactors? :-)

**Comment by cynthiameiring** [comment]
> Interface komt niet overeen met implementatie, last minute refactors? :-)

Fixed :+1:

File: src/Zicht/Bundle/SolrBundle/Manager/SolrEntityManager.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 namespace Zicht\Bundle\SolrBundle\Manager;
 
+use Zicht\Bundle\SolrBundle\Manager\DeleteIndexableRelationsInterface;
```
Comment (7ochem) [inline src/Zicht/Bundle/SolrBundle/Manager/SolrEntityManager.php:8]: Deze is niet nodig. Het is in dezelfde namespace

**Comment by cynthiameiring** [comment]
Do people have an opinion about whether this a patch or a minor release?

**Comment by wtfzdotnet** [comment]
I would suggest a minor as it's not a bug-patch :-).

**Review by 7ochem — APPROVED** [review]

**Review by 7ochem — CHANGES_REQUESTED** [review]
Weet niet precies wat je hebt gedaan, maar wil je je commits even squashen? :tennis:

**Review by 7ochem — APPROVED** [review]

**Pull request merged**